### ISSUE TITLE
Not rendering properly CSS in Ionic4 with Angular

### ISSUE THREAD

**Issue by eduboxgithub** [hodgef/simple-keyboard#132]
**Simple-keyboard version**
I'm using the latest version 2.23.2.

**Describe the bug**
I'm using Ionic 4 with Angular 7 to build a PWA.
I installed your great keyboard with npm and followed the Angular examples, but I'm struggling to make it work. The keyboard is instantiated, but it seems the CSS in not rendering properly or correctly loaded. 

What could be breaking the CSS?

I created a simple sample app, only with one page, where I try to render the keyboard.
You can download it and just make:

- npm install - install the dependencies
- ionic serve -  to build and run the local server

It will open a browser with the location: http://localhost:8000

[IonicSimpleKeyboard.zip](https://github.com/hodgef/simple-keyboard/files/3360311/IonicSimpleKeyboard.zip)

**Screenshots**
![image](https://user-images.githubusercontent.com/13800439/60684855-eaea8880-9e97-11e9-9465-484609b9673c.png)

**Comment by hodgef** [comment]
Hello @eduboxgithub,

Have you disabled `ViewEncapsulation` like in the following demo? If not, please give it a try:
https://codesandbox.io/s/github/hodgef/angular-simple-keyboard/tree/master/?fontsize=14

Regards,
Francisco Hodge

**Comment by eduboxgithub** [comment]
Hello @hodgef !

Thank you for your time and support.

I forgot to mention that I had tried that hypothesis, but I was getting an error when opening the application:

> ERROR Error: Uncaught (in promise): Error: Can not find module 'tslib'
Error: Can not find module 'tslib'

However, after your comment, I checked the code again and noticed that the automatic import of ViewEncapsulation, performed by VSCode, pointed to another location:

**Automatic import VSCode:** 
`import {ViewEncapsulation} from '@ angular / compiler / src / core';`

**Your Example:**
`import {ViewEncapsulation} from '@ angular / core';`

I fixed that, and I think it's working. =)

Thank you again for pointing me in the right direction.
